students/kuhnbt/lesson6/mailroom_unittests.py

```diff
@@ -0,0 +1,43 @@
+from mailroom_v4 import (get_thank_you, get_report_header, quit_program,
+                         get_report_row)
+import unittest
+
+
+class MailroomTest(unittest.TestCase):
+
+    def test_exit(self):
+        with self.assertRaises(SystemExit):
+            quit_program()
+
+    def test_thankyou(self):
+        hemingway_text = 'Dear Ernest Hemingway:\nThank you for your'\
+            ' generous donation of $40.00.\nI really appreciate your '\
+            '3\ndonations to our organization.\nI assure you that your'\
+            ' contributions will be put to\ngood use!\n\nRegards,\nBen'
+
+        self.assertEqual(hemingway_text, get_thank_you('Ernest Hemingway'))
```

Comment:
very nice!

a good idea is to test variations of `get_thank_you`, since you have special logic for "donations" vs "donation"

so you could have:
`test_get_thank_you_single_donation`
`test_get_thank_you_multiple_donations`

you should be able to over write `donor_info` with your test set of data to allow you to test various cases without compromising mailroom data set.
```
mailroom_v4.donor_info = defaultdict(list, {"Test Donor1": [1], "Test Donor2": [20, 24]})

```